heron/common/src/java/com/twitter/heron/common/basics/FileUtils.java

```diff
@@ -123,4 +123,17 @@ public static boolean isOriginalPackagePex(String packageFilename) {
   public static String getBaseName(String file) {
     return new File(file).getName();
   }
+
+  public static String getPkgType(String topologyBinaryFile) {
```

Comment:
Thanks for the removal of dup code!

Could you also please change to not abbreviate pkg (e.g., packageType and getPackageType).

Also instead of returning strings, better to use an enum, maybe as an inner enum in FileUtils:
```
enum PackageType {
  PEX,
  JAR,
  TAR
}
```